superset/assets/package.json

```diff
@@ -54,6 +54,11 @@
     "datatables-bootstrap3-plugin": "^0.5.0",
     "datatables.net": "^1.10.13",
     "datatables.net-bs": "^1.10.12",
+    "datatables.net-buttons": "^1.3.1",
+    "datatables.net-buttons-bs": "^1.3.1",
+    "font-awesome": "^4.6.3",
```

Comment:
I think we dropped this in favour of the fab shipped one.

Reply:
dt buttons?

Reply:
The comment is for font-awesome

Reply:
ah, I would be happy to remove the line provided it won't break anything. I'll do that now